src/main/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/KubernetesCredentialProvider.java

```diff
@@ -23,10 +23,7 @@
  */
 package com.cloudbees.jenkins.plugins.kubernetes_credentials_provider;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
NIT: the undocumented code style is to never import *.  
I should write that codestyle :)